### ISSUE TITLE
protocol for admins suggests that there is something special w.r.t. centre access

### ISSUE THREAD

**Issue by robertoostenveld** [Donders-Institute/donders-institute.github.io#17]
TBD: the following section is not really clear to me. Access permissions per collection are not related to the centers.

"This organization of the collections limits the access to these collections by members of other Donders centers or RU-external institutes, as the authorization for collection access is at the level of the collections (rather than at the center level). Authorization for the collection manager role, however, is at the center level."

I suspect that this text was related to an older plan for "centre viewers", which at this moment is not part of the initial implementation.

@EricMaris could you have a look at this? I wrote a TBD tag in the text. Note that I am currently working on it, so please don't do it immediately, but only after my commit of the updated text.

**Comment by EricMaris** [comment]
Corrected.

Main problem with he that text: the missing of the words "does not in any way". It had nothing to do with center viewers.
